getFunctions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @param array $shape The shape data for the required image.
+ *
+ * @return string The Img tag containing the shape and alternate text.
+ */
+
+function getNameImg(array $shape): string {
+    return "<img alt=\"" . $shape["ShapeName"] . "\"src=\"" . $shape["imgurl"] ."\" />";
+}
+
+/**
+ * @param array $shape The shape data for which the name is required
+ *
+ * @return string The name of the shape
+ */
+
+function getShapeName(array $shape): string {
```

Comment:
all of these get functions (except the image one) dont actually do anything, you can just remove them